notte/common/parser.py

```
@@ -86,8 +85,8 @@ def step(self, text: str) -> EnvStepParams:
         if action is None or not isinstance(action, str):
             raise ValueError("No action found")
         d = json.loads(action)
-        action_id = list(d.keys())[0]
-        params = d[action_id] if d[action_id] != {} else None
+        action_id = d["action-id"]
```

@TonioSnowden can you raise a value error if these keys are not present ?